### PR TITLE
test(#1625): cover ParallelTranslator auto threads, BytecodeClasses.all, annotation defaults, frames and matchers

### LINKED ISSUE #1625: Multiple uncovered code paths: ParallelTranslator, BytecodeClasses.all(), enum annotation values, defaults, frames
## What happens

Several important code paths have no direct coverage:

1. **`ParallelTranslator`** — no test proves real parallel execution or that `threads=0` (auto) works; only the negative `-1` validation is tested (`ParallelTranslatorTest`).
2. **`BytecodeClasses.all()`** — never tested against a real directory with `.class` files (only a non-existent dir and `toString()`).
3. **Enum annotation values** — `XmlAnnotationValue` branch `ENUM` (`XmlAnnotationValue.java:52-94`) is untested; only `DirectivesEnumAnnotationValue` (generation) is covered, not the parsing back.
4. **Annotation default values** — `BytecodeDefaultValue`/`XmlDefaultValue` (annotation method defaults) have no tests at all.
5. **`Format`** — `mode()`, `listing()`, `withListing()` and three `IllegalArgumentException` branches untested (`FormatTest`).
6. **Frames** — only `F_NEW` round-trip tested; `F_APPEND`/`F_CHOP`/`F_SAME`/`F_SAME1` and real frame values untested (`DirectivesFrameTest`).
7. **Matchers** — `SameXml`/`WithoutLines` have no negative cases (distinct XML must produce `false`).

## Expected

Cover the listed paths with focused unit tests (positive and negative where applicable).

## Files
- `src/test/java/org/eolang/jeo/ParallelTranslatorTest.java`
- `src/test/java/org/eolang/jeo/BytecodeClassesTest.java`
- `src/main/java/org/eolang/jeo/representation/xmir/XmlAnnotationValue.java:52-94`
- `src/test/java/org/eolang/jeo/representation/directives/FormatTest.java`
- `src/test/java/org/eolang/jeo/representation/directives/DirectivesFrameTest.java`
- `src/test/java/org/eolang/jeo/matchers/SameXmlTest.java`

### PR DESCRIPTION
**Pull request by VasilevNStas**
Fixes #1625

## Problem
Seven code paths had no direct coverage: `ParallelTranslator` parallel/auto execution, `BytecodeClasses.all()`
against a real directory, the `ENUM` branch of `XmlAnnotationValue`, annotation default values
(`BytecodeDefaultValue`/`XmlDefaultValue`), `Format` accessors and its `IllegalArgumentException` branches,
non-`F_NEW` frame types, and negative cases for `SameXml`/`WithoutLines`

## Solution / What
- `ParallelTranslatorTest` — `processesAllFilesWithAutoThreads` (threads=0) and `processesAllFilesWithPositiveThreads` (threads=2): 3 real `.class` files transformed to `.xmir` each
- `BytecodeClassesTest` — `findsAllClassFilesInDirectory`: a real tree with `.class` files and a `.txt` that must be ignored; `all()` count == 2
- `XmlAnnotationValueTest` — `createsEnumAnnotationProperty`: `DirectivesEnumAnnotationValue` → XMIR → `XmlAnnotationValue` → `BytecodeEnumAnnotationValue` round-trip
- `XmlDefaultValueTest` (new) — annotation method default value XMIR round-trip via `XmlDefaultValue.bytecode()`
- `FormatTest` — `mode()` (default `short` and explicit), `listing()`, `withListing()`, and the three `IllegalArgumentException` branches (non-boolean value, non-string value, odd property count, non-string property name)
- `DirectivesFrameTest` — parameterized `convertsDifferentFrameTypes` for `F_APPEND`/`F_CHOP`/`F_SAME`/`F_SAME1`
- `SameXmlTest` / `WithoutLinesTest` — negative cases (distinct XML → `false`; transformed XML contains no `line=`)

## Changes
- 1 new test file; 7 existing test files extended

## Tests
- 33 focused tests pass; full `mvn clean install -Pqulice` green (qulice + jtcop + jacoco)

### CODE REVIEW

**Comment by VasilevNStas** [comment]
@yegor256 
plz review

File: src/test/java/org/eolang/jeo/ParallelTranslatorTest.java [hunk 1]
```diff
@@ -104,6 +104,54 @@ void assemblesSuccessfully(@TempDir final Path temp) throws IOException {
...
+            .apply(
+                Stream.of(1, 2, 3)
+                    .map(index -> temp.resolve(String.format("Class%d.class", index - 1)))
+            )
+            .collect(Collectors.toList());
+        for (int index = 0; index < 3; ++index) {
+            MatcherAssert.assertThat(
+                String.format("Class%d must be transformed with the given thread count", index),
+                temp.resolve(String.format("Class%d.xmir", index)).toFile(),
+                FileMatchers.anExistingFile()
```
Comment (zaguzovmaksim0-hue) [inline src/test/java/org/eolang/jeo/ParallelTranslatorTest.java:150]: This proves `threads=2` processes all inputs, but it does not prove the issue's stated missing behavior: real parallel execution. A sequential implementation that simply loops over the stream would still create all three `.xmir` files and pass this test. Please make the transformation record/coordinate concurrent overlap (for example with latches/barriers or an active/max-active counter) and assert max concurrency > 1 for the positive-thread case, while keeping the auto-thread case focused on successful execution.

**Review by zaguzovmaksim0-hue — CHANGES_REQUESTED** [review]
Most uncovered paths in #1625 are now exercised, but the ParallelTranslator requirement is still only a completeness test. The issue specifically asks for proof of real parallel execution; the current `threads=2` test would pass a sequential implementation. Please assert actual overlap/concurrency.
